glommio/src/shares.rs

```diff
@@ -7,6 +7,8 @@ use core::fmt::Debug;
 use std::rc::Rc;
 use std::time::Duration;
 
+pub(crate) const MAX_SHARES: u64 = 1u64 << 22;
```

Comment:
This is the only actual code change in this PR; I moved a constant used in two places out to a named one.

This could probably use a better name and a comment, but I'm not sure I understand it well enough to add myself.

Reply:
This isn't `MAX_SHARES`.

I don't mind a constant here, but this is really just a math shift. The role of this is to turn a small quantity big so we don't have to resort to float math nor lose precision. When the reciprocal shares are turned to shares there is another shift that takes some of those bits away.

In any case I would prefer you would rebase the tree and remove this patch: it is a documentation PR so it's best and less confusing if it only touch documentation. Happy to tackle this later separately.